src/DTO/Section/Section.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Petersons\D2L\DTO\Section;
+
+use Illuminate\Support\Collection;
+use Petersons\D2L\DTO\RichText;
+
+final class Section
+{
+    public function __construct(
+        public int        $sectionId,
```

Comment:
@jolezg All properties should be private to be consistent with other DTOs. We don't need public properties and public getters, especially since public non read only properties break encapsulation.